isisd/isis_nb_config.c

```diff
@@ -1508,6 +1522,96 @@ int isis_instance_segment_routing_srgb_upper_bound_modify(
 	return NB_OK;
 }
 
+/*
+ * XPath: /frr-isisd:isis/instance/segment-routing/srlb
+ */
+int isis_instance_segment_routing_srlb_pre_validate(
```

Comment:
I leave the pre-validation separate as 1) it is hard to determine if we are in SRGB or SRLB node to get respectively the SRLB or SRGB value from the yang tree. And also because I add the verification of size lower than 65535

Reply:
Ok. Having a single `pre_validate()` callback was mostly a matter of reducing code duplication (the SRGB/SRLB overlapping ranges check), but having separate callbacks is fine too.